src/main/java/com/everylingo/everylingoapp/controller/RequestController.java

```diff
@@ -0,0 +1,50 @@
+package com.everylingo.everylingoapp.controller;
+
+import com.everylingo.everylingoapp.model.TranslationRequest;
+import com.everylingo.everylingoapp.repository.TranslationRequestRepository;
+import com.everylingo.everylingoapp.service.TranslationRequestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class RequestController {
+    @Autowired
+    private TranslationRequestService translationRequestService;
+
+    @Autowired
+    private TranslationRequestRepository translationRequestRepository;
+
+    @GetMapping("/translationrequest")
+    private List<TranslationRequest> getAllRequests()
+    {
+        return translationRequestService.getAllRequests();
+    }
+
+    @GetMapping("/translationrequest/{translationrequestid}")
+    private TranslationRequest getRequest(@PathVariable("translationrequestid") int translationRequestid)
```

Comment:
Since we are working with OAuth 2.0 authorization, you should adapt all controller methods to take in @AuthenticationPrincipal OAuth2User oAuth2User as parameter